scripts/gallery/numpy/team.md

```diff
@@ -0,0 +1,141 @@
+## NumPy Team Gallery
+
+We are an international team on a mission to support scientific and research communities worldwide by building quality, open-source software. [**Join us**](/contribute)!
+
+<hr>
+
+###Maintainers
+
+<div class="row maintainers">
+	{% for row in main.maintainers.people | batch(9, "") %}
+		<div class="card-group maintainers">
+		{% for person in row %}
+			{% if person %}
+				<div class="card rounded">
+					<img class="card-img-top" style="border-radius: 5%;" alt="" src="{{ person.avatar_url }}"/>
+					<div class="card-body">
+						<h6 class="card-title">
+							{% if person.blog %}
```

Comment:
Design wise, I'd prefer to remove these links. Rationale:
- Some people have links and some don't, which means tiles have either black or blue text.
- The links are fairly random, sometimes it's LinkedIn, sometimes a personal blog, or yet something else. 
- Those links can easily be found some other way

Reply:
Should we use the location information in Github instead of blog?  Or just names?

Reply:
In gh-89 there was a suggestion to use a map to show everyone's location. However that's quite a bit of work and a separate issue; for now I would suggest just to use the names.

Reply:
> Design wise, I'd prefer to remove these links. Rationale:
> 
> * Some people have links and some don't, which means tiles have either black or blue text.
> * The links are fairly random, sometimes it's LinkedIn, sometimes a personal blog, or yet something else.
> * Those links can easily be found some other way

Done.